src/gh_pr/utils/notifications.py

```
@@ -0,0 +1,369 @@
+            # Add urgency for notify-send
+            if self._notifier == 'notify-send':
+                cmd[2] = f'--urgency={urgency}'
+
+                # Add timeout
+                cmd.extend(['-t', str(self.config.timeout * 1000)])
+
+                # Add icon if available
+                if self.config.icon_path and self.config.icon_path.exists():
+                    cmd.extend(['-i', str(self.config.icon_path)])
+
+            result = subprocess.run(
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)